website/src/controllers/api-v2/groups.js

```diff
@@ -691,8 +691,9 @@ var inviteByEmails = function(invites, group, req, res, next){
 
 api.invite = function(req, res, next){
   var group = res.locals.group;
-
-  if (req.body.uuids) {
+  if (group.memberCount + group.invites.length >= 30) {
+    return res.json(400, {err: "Over 30 member party cap"});
```

Comment:
@colwem you can add a variable in a string like it's done here https://github.com/HabitRPG/habitrpg/blob/develop/common/locales/en/challenge.json#L41, then to pass the value you'll do `env.t('stringName', {varName: 30})`